core/src/main/scala/kafka/log/ProducerStateManager.scala

```diff
@@ -939,3 +940,24 @@ object SnapshotFile {
     SnapshotFile(file, offset)
   }
 }
+
+class ProducerStateManagerConfig(@volatile var producerIdExpirationMs: Int) extends Logging with BrokerReconfigurable {
+
+  override def reconfigurableConfigs: Set[String] = Set(KafkaConfig.ProducerIdExpirationMsProp)
```

Comment:
Should we use `ProducerStateManagerConfig.ReconfigurableConfigs` to be consistent?